mabar-frontend/src/services/aiMatchmakingService.ts

```
@@ -0,0 +1,456 @@
+          break
+        
+        case 'checkBookingStatus':
+          response = await MatchmakingToolboxService.checkBookingStatus(parameters)
+          break
+        
+        case 'joinSession':
+          response = await MatchmakingToolboxService.joinSession(parameters)
+          break
+        
+        case 'deleteBooking':
+          response = await MatchmakingToolboxService.deleteBooking(parameters)
```

Biome (2.1.2)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)